### PR TITLE
refactor(handlers/guild): rename guild widget functions

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**
Since Discord doesn't call guild Embeds embed but widget functions should have the same name to avoid confusion.

Note: This is a breaking change don't merge this before v11!
**Status**

- [ ] Code changes have been tested against the Discord API, or there are no code changes

**Semantic versioning classification:**

- [x] This PR changes the library's interface (methods or parameters added)
  - [x] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by itohatweb** [comment]
Closed the wrong one.

File: src/api/handlers/guild.ts [hunk 1]
```diff
@@ -455,8 +455,8 @@ export async function getAuditLogs(
   });
 }
 
-/** Returns the guild embed object. Requires the MANAGE_GUILD permission. */
-export async function getEmbed(guildID: string) {
+/** Returns the guild widget object. Requires the MANAGE_GUILD permission. */
+export async function getWidgetSettings(guildID: string) {
```
Comment (ayntee) [inline src/api/handlers/guild.ts:459]: Better name? How about just `getWidget`?

**Review by ayntee — APPROVED** [review]
LGTM :heart:

File: src/api/handlers/guild.ts (hunk 1 above)
Comment (itohatweb) [reply src/api/handlers/guild.ts:459]: This Endpoint returns the settings for the Widget. There is an other Endpoint hat returns the Widget so naming this one getWidget makes more sense to me.

**Review by Skillz4Killz — APPROVED** [review]

**Review by Skillz4Killz — APPROVED** [review]

**Review by ayntee — APPROVED** [review]

**Pull request merged**